client/lib/layout-focus/index.js

```diff
@@ -34,33 +49,25 @@ var layoutFocus = {
 	},
 
 	set: function( area ) {
-
-		if ( ! this.isValid( area ) || area === this._current ) {
+		if ( ! isValid( area ) || area === this._current ) {
```

Comment:
Minor: Reversing the order here would allow the condition to be shortcut in the case that the area hasn't changed. Reference comparison [more performant](https://albertxing.com/stool/#734fd0b246ce057385a0400e0de7ee17) than `Array#indexOf` lookup in `isValid` and is probably the more likely reason for this function to be aborted:

```js
if ( area === this._current || ! isValid( area ) ) {
```